StackExRedis/NHibernate.Caches.StackExRedis/DefaultConnectionMultiplexerProvider.cs

```diff
@@ -13,7 +13,6 @@ public IConnectionMultiplexer Get(string configuration)
 		{
 			TextWriter textWriter = Log.IsDebugEnabled() ? new NHibernateTextWriter(Log) : null;
 			var connectionMultiplexer = ConnectionMultiplexer.Connect(configuration, textWriter);
-			connectionMultiplexer.PreserveAsyncOrder = false; // Recommended setting
```

Comment:
This is obsolete with 2.0, and is false by default (see [here](https://stackexchange.github.io/StackExchange.Redis/ReleaseNotes#20495) for more info).